test/test_sparse.py

```diff
@@ -2603,6 +2603,16 @@ def sparse_log(x):
                 ly1 = torch.sparse.log_softmax(x, dim)
                 self.assertEqual(ly1, sparse_log(y1))
 
+                if torch.cuda.is_available():
```

Comment:
The outer `test_softmax` here is still decorated with `@cpu_only`, looks like that should be removed?

Reply:
In the future it'd be nice if test_sparse.py added a device generic test class. Then tests could run on both CPU and CUDA.